### DISCUSSION TITLE
Corepack's yarn and pnpm presist paths.

### DISCUSSION THREAD

**Discussion by marknefedov** [ScoopInstaller/Main#4384] [General]
Should scoop configure pnpm and yarn to store its data in persist folder too?
Something like this.
```
corepack enable
pnpm config set global-dir presist-dir
yarn config set global-folder presist-dir
corepack disable
```
Currently, they are scattered all over the place.
![image](https://user-images.githubusercontent.com/20001291/214825526-13683dcc-5fb2-47a2-8120-adcc5b2e3899.png)
![image](https://user-images.githubusercontent.com/20001291/214825554-9e5c8929-0692-4f7e-b821-427c9ddd55cd.png)

**Comment by rashil2000** [comment]
Yeah, seems like a good idea to me
